src/components/PageHeader/PageHeader.tsx

```diff
@@ -54,7 +54,7 @@ export const PageHeader = ({
   );
 
   return (
-    <div className={componentClassName} {...other}>
+    <header className={componentClassName} {...other}>
```

Comment:
Not technically within the scope of this ticket, but as I was auditing our components I noticed that this should be using <header> instead of <div>.